pkg/kube/inject/webhook.go

```diff
@@ -310,6 +310,10 @@ func getInjectionStatus(podSpec corev1.PodSpec) string {
 	for _, c := range podSpec.ImagePullSecrets {
 		stat.ImagePullSecrets = append(stat.ImagePullSecrets, c.Name)
 	}
+	if revision == "" {
+		revision = "default"
+	}
+	stat.Revision = revision
```

Comment:
can you add some comments on why we did this? About tags vs revision specifically?

A year from now it will probably not be obvious why it matters

Reply:
Good call, added